examples/force_handle_mixin.py

```diff
@@ -0,0 +1,94 @@
+"""
+Create Pipeline Steps that only require implementing handler methods.
+========================================================================================================================
+
+If a pipeline step only needs to implement handler methods, then you can inherit from the ForceHandleMixin as demonstrated here.
+Handler methods are useful when :
+- You need to change the shape of the data container passed to the following steps, or the wrapped steps.
+- You want to apply side effects based on the data container, and the execution context.
+- You want to change the pipeline execution flow based on the data container, and the execution context.
+
+..
+    Copyright 2019, Neuraxio Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+..
+    Thanks to Umaneo Technologies Inc. for their contributions to this Machine Learning
+    project, visit https://www.umaneo.com/ for more information on Umaneo Technologies Inc.
+
+"""
+import numpy as np
+
+from neuraxle.base import BaseStep, ForceHandleMixin, DataContainer, ExecutionContext
+from neuraxle.pipeline import Pipeline
+
+
+class ForceHandleMixinStep(ForceHandleMixin, BaseStep):
+    """
+    Please make your steps inherit from ForceHandleMixin, when they only implement handle_methods.
```

Comment:
No `,` after `ForceHandleMixin`.

Reply:
Also say that "when they only implement handle_methods `but that you make impossible the use of regular fit, transform, and fit_transform methods`".